lib/ansible/modules/network/nxos/nxos_file_copy.py

```diff
@@ -56,13 +56,20 @@
         their default values.
     required: false
     default: null
+  connect_port:
```

Comment:
naming this as `connect_ssh_port` will be imply use of ssh port on the server.

Reply:
@kedarX resolved.